FitnessClub/src/app/core/service/auth.service.ts

```diff
@@ -1,67 +1,66 @@
 import { Injectable } from '@angular/core';
-import { environment } from "../../../environments/environment";
-import {HttpClient} from "@angular/common/http";
-import {StorageService} from "./storage.service";
-import {SignInModel} from "../models/sign-in.model";
-import {catchError, map} from 'rxjs/operators';
-import {Observable, of, throwError} from 'rxjs';
-import { Register} from "../models/register.model";
-import {UserIdModel, UserRoleModel} from '../models/UserId.model';
-import { Router } from '@angular/router';
-import {TokenResponse} from "../models/TokenResponse.model";
+import { environment } from '../../../environments/environment';
+import { HttpClient } from '@angular/common/http';
+import { StorageService } from './storage.service';
+import { SignInModel } from '../models/sign-in.model';
+import { catchError, map } from 'rxjs/operators';
+import { Observable, of } from 'rxjs';
+import { Register } from '../models/register.model';
+import { UserIdModel, UserRoleModel } from '../models/UserId.model';
+import { TokenResponse } from '../models/TokenResponse.model';
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class AuthService {
-
   userId: string | undefined;
 
   private readonly apiUrl: string = `${environment.apiIdentityAddress}/Authenticate`;
   private readonly apiUrlReg: string = `${environment.apiIdentityAddress}/Registration`;
-  private  readonly apiNgrok: string = `${environment.apiNgrok}/Registration`;
-  private  readonly apiApi: string = `${environment.apiAddress}/User`;
-  constructor(private readonly httpClient: HttpClient,
-              private readonly storageService: StorageService,
-              private router: Router
-              ) { }
+  private readonly apiApi: string = `${environment.apiAddress}/User`;
+
+  constructor(
+    private readonly httpClient: HttpClient,
+    private readonly storageService: StorageService
+  ) {}
+
   signIn(signInModel: SignInModel): Observable<any> {
-    return this.httpClient.post<any>(`${this.apiUrl}/login`, signInModel)
-      .pipe(
```

Comment:
this code is doing pretty much nothing
maps the response into itself
also catches an error and throws it further
so you can easily remove it and simplify the code